web/src/app/core/services/monitor.service.ts

```diff
@@ -43,9 +43,9 @@ export class MonitorService {
           if (monitors.find((monitorModel: MonitorModel) => monitorModel.uid === monitor.uid)) {
             return project.monitors.map((monitorModel: MonitorModel) => {
               if (monitorModel.uid === monitor.uid) {
+                this.pingMonitor(projectUid, monitor.uid, 'automatic');
```

Comment:
I just noticed this needs to be returned in a the `pipe` because its an observable

Reply:
this one was a bit interesting, I have had a play with this and will push up my changes for you to review